capstone/src/main/java/com/google/sps/Objects/Group.java

```diff
@@ -19,22 +19,31 @@
   private final String groupName;
   private final String headerImg;
   private final long groupId;
+  private double midLatitude;
+  private double midLongitude;
+  private ArrayList<Long> locationIds;
 
   public Group(
       ArrayList<String> memberIds,
       ArrayList<Long> challenges,
       ArrayList<Long> posts,
       ArrayList<Long> options,
+      ArrayList<Long> locationsIds,
```

Comment:
Could you explain to me why you need this? At first glance it appears as though the intent is to store the list of user locations for a group. However, looking at GroupLocation.java, I see that you use the userIDs of a group to get a user from the DB and then get their location, so this list of location IDs seems irrelevant?
Also, I don't see any of the servlets that you edited do anything with adding a user to a group so I don't see this list of locations being created or used?

Reply:
I'm using this list to store the Id's for the nearest found locations/restaurants for a group. The Location object has the address, name, lat/lng properties to store location information.